core/src/core-api/_internalLinkManager.js

```diff
@@ -54,6 +54,30 @@ export class linkManager extends LuigiCoreAPIBase {
     return remotePromise;
   }
 
+  navigateToIntent(semanticSlug, params = {}) {
```

Comment:
Can you add a JSDoc comment header for this function as well. I know its written somewhere else too, but better have it redundant here as well.